front/app/containers/Admin/projects/project/phase/phaseParticipationConfig/components/messages.ts

```diff
@@ -37,6 +37,15 @@ export default defineMessages({
     defaultMessage:
       "Tap into your users' collective intelligence. Invite them to submit, discuss ideas, and/or provide feedback in a public forum.",
   },
+  proposalsTitle: {
+    id: 'app.containers.AdminPage.phase.methodPicker.proposalsTitle',
+    defaultMessage: 'Proposals, petitions or initiatives',
+  },
+  proposalsDescription: {
+    id: 'app.containers.AdminPage.phase.methodPicker.proposalsDescription',
+    defaultMessage:
+      'Allow participants to upload ideas with a time and vote limit.',
```

Comment:
I think if I were selecting a participation method, I might not fully understand the different between ideation and proposals/initiatives with this copy? :thinking:

Reply:
Or maybe that's just me :) I don't have any good suggestions either really.

Reply:
Indeed, good point! I think we might need to work on the copy a bit before exposing this to users but for now, it's all behind a feature flag so it can remain as is before we can have a better idea.